mu-plugins/blocks/global-header-footer/js/wporg-global-header-script.js

```diff
@@ -188,4 +178,15 @@
 			} );
 		} );
 	} );
+
+	/* eslint-disable @wordpress/no-global-event-listener */
+	window.addEventListener( 'resize', () => {
+		// Hide any open mobile menus if we're no longer in a mobile view.
+		if ( ! document.querySelector('.global-header__navigation .wp-block-navigation__responsive-container-open').offsetWidth ) {
+			let closeMenuButton = document.querySelector('.wp-block-navigation__responsive-container.is-menu-open [data-micromodal-close]');
```

Comment:
```suggestion
			const closeMenuButton = document.querySelector( '.wp-block-navigation__responsive-container.is-menu-open button[data-micromodal-close]' );
```

For some reason this library adds that `data-micromodal-close` attr to the modal _and_ the button, so this just ensures you're grabbing the button. Both work, but specifying the button would probably be more stable.

Reply:
@ryelle Agreed, I committed your suggestion as-is, I had originally targeted the button, but swapped to the attribute and failed to notice that it added it on both elements when it still worked.